src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreEditor.res

```diff
@@ -5,7 +5,6 @@
 @val external innerHeight: int = "window.innerHeight"
 
 @val external document: 'a = "document"
-%%raw("import './styles.css'")
```

Comment:
@Pragati1610 @rithviknishad 
removing this is causing styling issues like when there is no highlighting based on push score

Reply:
Figuring out why that is happening when the styles in that file is just applied on a checkbox and it has been accounted for in the global css stylesheet

Reply:
can you confirm if it is still happening @khavinshankar